ckanext/validation/helpers.py

```diff
@@ -4,6 +4,18 @@
 from ckan.lib.helpers import url_for_static
 from ckantoolkit import url_for, _, config, asbool, literal, h
 
+def _get_helpers():
```

Comment:
underscore syntax for python implies that the method should not be imported into another module, but this is used in plugin.py for get_helpers.

change to match the rest of the methods. E.g. `get_actions` and `get_blueprints`. So plugins.py imports `ckanext.validation.helpers` and does `helpers.get_helpers`